partition/roles/mgmt-server/tasks/main.yaml

```diff
@@ -85,9 +87,66 @@
   notify:
     - reload frr
 
+- name: Copy ssh key pair for metal user
+  copy:
+    content: "{{ item.content }}"
+    dest: "{{ item.dest }}"
+    mode: "{{ item.mode}}"
+    owner: metal
+    group: metal
+  loop:
+  - content: "{{ mgmt_server_metal_ssh_privkey }}"
+    dest: /home/metal/.ssh/id_rsa
+    mode: "0600"
+  - content: "{{ mgmt_server_metal_ssh_pubkey }}"
+    dest: /home/metal/.ssh/id_rsa.pub
+    mode: "0644"
```

Comment:
Why we need the public key on the machine?

Reply:
I don't know.